applications/ShallowWaterApplication/python_scripts/postprocess/generate_gid_list_file.py

```diff
@@ -0,0 +1,64 @@
+from os import path, listdir, getcwd
```

Comment:
I recommend checking out the [pathlib](https://docs.python.org/3/library/pathlib.html) lib which makes those things easier / more native

Reply:
pathlib is really nice! Thanks